src/SFML/Audio/Music.cpp

```diff
@@ -109,36 +110,171 @@ Time Music::getDuration() const
 
 
 ////////////////////////////////////////////////////////////
+Uint64 Music::getSampleCount() const
+{
+    return m_file.getSampleCount();
+}
+
+
+////////////////////////////////////////////////////////////
+Music::TimeSpan Music::getLoopPoints() const
+{
+    TimeSpan out(Time::Zero, Time::Zero);
+
+    // Make sure we don't divide by 0
```

Comment:
Divide by zero should better be checked where the divide is done, i.e. in samplesToTime. Which makes this function much less verbose:

```cpp
return TimeSpan(samplesToTime(m_loopSpan.offset), samplesToTime(m_loopSpan.length));
```